pkg/controller/statefulset/stateful_set_test.go

```diff
@@ -510,6 +509,8 @@ func TestGetPodsForStatefulSetAdopt(t *testing.T) {
 	pod4.OwnerReferences = nil
 	pod4.Name = "x" + pod4.Name
 
+	ssc, spc := newFakeStatefulSetController(set, pod1, pod2, pod3, pod4)
```

Comment:
Why changing this? Or is this included by accident?

Reply:
I added the Patch implementation for fakeKubeClient and suddenly a lot of tests failed. The fake Patch used to has no effect and just return success no matter what. Now the fake Patch works. This unit test needs to be fixed because of this.